src/portal/report.md

```diff
@@ -0,0 +1,48 @@
+# Azure CLI Module Creation Report
+
+### portal dashboard create
+
+create a portal dashboard.
+
+|Option|Type|Description|Path (SDK)|Path (swagger)|
+|------|----|-----------|----------|--------------|
+|**--resource_group_name**|string|The name of the resource group.|resource_group_name|resource_group_name|
+|**--name**|string|The name of the dashboard.|dashboard_name|name|
+|**--location**|string|Resource location|location|location|
+|**--tags**|dictionary|Resource tags|tags|tags|
+|**--lenses**|dictionary|The dashboard lenses.|lenses|properties_lenses|
+|**--metadata**|dictionary|The dashboard metadata.|metadata|properties_metadata|
+### portal dashboard delete
+
+delete a portal dashboard.
+
+|Option|Type|Description|Path (SDK)|Path (swagger)|
+|------|----|-----------|----------|--------------|
+|**--resource_group_name**|string|The name of the resource group.|resource_group_name|resource_group_name|
+|**--name**|string|The name of the dashboard.|dashboard_name|name|
+### portal dashboard list
+
+list a portal dashboard.
```

Comment:
SHould we have import in here?

Reply:
> SHould we have import in here?

idk where this file in use, but it looks we should add "import" here as well.

Reply:
well, this file may be entirely removed. it's an extra.